backend/de.metas.cucumber/src/test/resources/de/metas/cucumber/features/pricing/scalePrices.feature

```diff
@@ -0,0 +1,107 @@
+@from:cucumber
+Feature: scale prices
+
+  Background:
+    Given the existing user with login 'metasfresh' receives a random a API token for the existing role with name 'WebUI'
+    And metasfresh has date and time 2022-05-17T13:30:13+01:00[Europe/Berlin]
+
+    And metasfresh contains M_PricingSystems
+      | Identifier           | Name                 | Value                |
+      | defaultPricingSystem | defaultPricingSystem | defaultPricingSystem |
+    And metasfresh contains M_Products:
+      | Identifier             | Name                   |
+      | salesProduct_180520225 | salesProduct_180520225 |
+    And metasfresh contains M_PriceLists
+      | Identifier       | M_PricingSystem_ID.Identifier | OPT.C_Country.CountryCode | C_Currency.ISO_Code | Name           | SOTrx | IsTaxIncluded | PricePrecision |
+      | defaultPriceList | defaultPricingSystem          | DE                        | EUR                 | PriceListName1 | true  | false         | 2              |
+    And metasfresh contains M_PriceList_Versions
+      | Identifier | M_PriceList_ID.Identifier | Name     | ValidFrom  |
+      | defaultPLV | defaultPriceList          | PLVName1 | 2022-05-11 |
+    And metasfresh contains C_BPartners:
+      | Identifier | Name         | OPT.IsCustomer | M_PricingSystem_ID.Identifier |
+      | bpartner_1 | BPartnerTest | Y              | defaultPricingSystem          |
+
+  @from:cucumber
+  Scenario: scale price with 'Use scale price' set on product price
+    Given metasfresh contains M_ProductPrices
+      | Identifier | M_PriceList_Version_ID.Identifier | M_Product_ID.Identifier | PriceStd | C_UOM_ID.X12DE355 | C_TaxCategory_ID.InternalName | OPT.UseScalePrice |
+      | pp_1       | defaultPLV                        | salesProduct_180520225  | 5.00     | PCE               | Normal                        | S                 |
+    And metasfresh contains M_ProductScalePrices:
+      | Identifier | M_ProductPrice_ID | PriceLimit | PriceList | PriceStd | Qty |
+      | ps_1       | pp_1              | 0          | 0         | 5        | 1   |
+      | ps_2       | pp_1              | 0          | 0         | 4.5      | 10  |
+      | ps_3       | pp_1              | 0          | 0         | 4        | 100 |
+    And metasfresh contains C_Orders:
+      | Identifier | IsSOTrx | C_BPartner_ID.Identifier | DateOrdered |
+      | o_1        | true    | bpartner_1               | 2022-05-17  |
+    And metasfresh contains C_OrderLines:
+      | Identifier | C_Order_ID.Identifier | M_Product_ID.Identifier | QtyEntered |
+      | ol_1       | o_1                   | salesProduct_180520225  | 8          |
+      | ol_2       | o_1                   | salesProduct_180520225  | 15         |
+      | ol_3       | o_1                   | salesProduct_180520225  | 150        |
+    When update prices for order line
```

Comment:
sooo...why do we need this step? doesn't that mean that the  scale prices would not be applied in the real world either?
I would have thought that there is some ModelInterceptor in chanrge of updating the order-line-prices automatically 🤔

Reply:
good point, the ones I saw were callouts but truth to be told, I didn't check all of them... so @mi-ps pls check if the price calculator will be called for a line created in cucumber

Reply:
LE: so, i would like ask you to look into changing those callouts into MIs

Reply:
@metas-ts @cp-ps yes, you are right, the updateOrderLine method gets called in the process of creating a new order line. 
So there is a method org.compiere.model.MOrderLine#setPrice() that eventually calls de.metas.order.impl.OrderLinePriceCalculator#updateOrderLine() and that gets called before saving the orderline as well as from various interceptors. The interceptor that we found in this case is de.metas.order.model.interceptor.C_OrderLine#updatePricesOverrideExistingDiscounts()